common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain grading policy"),
+                        "long_description": _(
+                            "State the course grading policy and certificate requirements clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain learning objectives"),
+                        "long_description": _(
+                            "State the learning objectives, goals, and course outcome(s) clearly on the course syllabus."
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain academic policy"),
+                        "long_description": _(
+                            "State the academic policy for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain forum participation expectations"),
+                        "long_description": _(
+                            "State the expectation for learner forum participation for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Provide forum etiquette guidelines"),
+                        "long_description": _(
+                            "State the guidelines for forum use and ediquette for the course on the course syllabus"
+                        ),
+                        "is_checked": False,
+                        "action_url": "",
+                        "action_text": "",
+                        "action_external": False,
+                    },
+                    {
+                        "short_description": _("Explain instructor/TA involvement"),
+                        "long_description": _(
+                            "Explain the role and level of involvement that instructors and course TAs for the course on the course syllabus"
```

Comment:
Please use "instructors" instead of "faculty," since that's a more academic term